packages/base-styles/_z-index.scss

```diff
@@ -31,12 +31,11 @@ $z-layers: (
 	".interface-interface-skeleton__header": 30,
 	".interface-interface-skeleton__content": 20,
 	".edit-widgets-header": 30,
-	".wp-block-cover__inner-container": 1, // InnerBlocks area inside cover image block.
 	".wp-block-cover.is-placeholder .components-placeholder.is-large": 1, // Cover block resizer component inside a large placeholder.
-	".wp-block-cover.has-background-dim::before": 1, // Overlay area inside block cover need to be higher than the video background.
-	".block-library-cover__padding-visualizer": 2, // BoxControl visualizer needs to be +1 higher than .wp-block-cover.has-background-dim::before
-	".wp-block-cover__image-background": 0, // Image background inside cover block.
-	".wp-block-cover__video-background": 0, // Video background inside cover block.
+	".wp-block-cover.has-background-dim::before": 0, // Overlay area inside block cover need to be higher than the video background.
+	".block-library-cover__padding-visualizer": 1, // BoxControl visualizer needs to be +1 higher than .wp-block-cover.has-background-dim::before
```

Comment:
TODO: Check where it's applied and test.

Reply:
Unfortunately, I couldn't find where it's used. I tried to search for "padding-visualizer" in the HTML of the editor, the frontend, and the Gutenberg codebase, but I didn't find anything. Anyway, I'm following the rule suggested in the comment, where it's _"+1 higher than .wp-block-cover.has-background-dim::before"_.

Reply:
Removed here as suggested: https://github.com/WordPress/gutenberg/pull/65626/commits/5a47d854a7890189a96df5c0d1dd27fe91cfe26c